### ISSUE TITLE
[Proposal] Analyzer that suggests ToUpperInvariant/ToLowerInvariant in place of ToUpper/ToLower

### ISSUE THREAD

**Issue by jamesqo** [dotnet/roslyn-analyzers#1487]
Something similar is already in place here, but for string comparisons: https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.NetCore.Analyzers/Core/Runtime/UseOrdinalStringComparison.cs.

**Comment by TKharaishvili** [comment]
@mavasani created a PR for this - https://github.com/dotnet/roslyn-analyzers/pull/2186

**Comment by carlossanlop** [comment]
@mavasani should this go through API review in the runtime repo, or is this just an improvement to an existing analyzer?

**Comment by mavasani** [comment]
Yes, I agree we need a proposal/triage in runtime repo before taking this forward.
